node/src/p2p/p2p_effects.rs

```diff
@@ -474,12 +464,7 @@ pub fn node_p2p_effects<S: Service>(store: &mut Store<S>, action: P2pActionWithM
                                     work: snark.clone(),
                                 });
                             }
-                            Some(P2pRpcResponse::InitialPeers(peers)) => {
-                                store.dispatch(P2pDiscoveryAction::Success {
-                                    peer_id,
-                                    peers: peers.iter().cloned().collect(),
-                                });
-                            }
+                            Some(P2pRpcResponse::InitialPeers(_)) => {}
```

Comment:
Do we still need this?

Reply:
I think it is not need now, but it might be needed by webnode later, @binier is it so?